UnitTests/BraintreeCoreTests/Analytics/FPTIBatchData_Tests.swift

```diff
@@ -99,6 +101,8 @@ final class FPTIBatchData_Tests: XCTestCase {
         XCTAssertNil(eventParams[1]["error_desc"])
         XCTAssertEqual(eventParams[0]["correlation_id"] as? String, "fake-correlation-id-1")
         XCTAssertNil(eventParams[1]["correlation_id"])
+        XCTAssertEqual(eventParams[0]["is_vault"] as? Bool, true)
+        XCTAssertNil(eventParams[1]["is_vault"])
```

Comment:
It looks like this is failing currently, should be:
```suggestion
        XCTAssertEqual(eventParams[0]["is_vault"] as? Bool, false)
        XCTAssertEqual(eventParams[1]["is_vault"] as? Bool, true)
```